MTFO/Managers/ConfigManager.cs

```diff
@@ -43,12 +43,17 @@ static ConfigManager()
             GAME_VERSION = GetGameVersion();
 
             GameDataPath = ResolveGameDataPath(Path.Combine(Paths.BepInExRootPath, "GameData"));
-            if (string.IsNullOrEmpty(GameDataPath)) GameDataPath = ResolveGameDataPath(Paths.PluginPath);
+            if (string.IsNullOrEmpty(GameDataPath))
+            {
+                GameDataPath = ResolveGameDataPath(Paths.PluginPath);
+                Log.Warn("Plugin paths for gamedata are under legacy support and will be removed in the future. Considering migrating to the '\\BepInEx\\GameData' folder.");
+            }
 
             if (HasGameDataPath)
             {
                 CustomPath = Path.Combine(GameDataPath, CUSTOM_FOLDER);
                 HasCustomContent = Directory.Exists(CustomPath);
+                Log.Warn("Consider migrating custom json data to the '\\BepInEx\\GameData folder instead");
```

Comment:
This warning will be thrown for either path usage. Maybe add a check to make sure that CustomPath is targeting legacy.

Reply:
The intent is to nudge plugin developers away from using the MTFO custom path, since the bepinex gamedata path

Reply:
Removed warning instead